py_clob_client/client.py

```diff
@@ -322,6 +339,44 @@ def create_order(
             ),
         )
 
+    def create_market_order(
+        self, order_args: MarketOrderArgs, options: PartialCreateOrderOptions = None
+    ):
+        """
+        Creates and signs an order
+        Level 1 Auth required
+        """
+        self.assert_level_1_auth()
+
+        # add resolve_order_options, or similar
+        tick_size = self.__resolve_tick_size(
+            order_args.token_id,
+            options.tick_size if options else None,
+        )
+        neg_risk = options.neg_risk if options else False
+
+        if order_args.price is None or order_args.price <= 0:
+            p = self.get_price(order_args.token_id, "SELL")
```

Comment:
so we just use best bid? why always "sell"?

Reply:
why is price even needed here ig?

Reply:
when you use `SELL` as param of the `get_price` function/request, you get the price of the best ask available and that is what we want to match, right? we want to BUY the best ASK available.

In the clob we have:
```go
func (b *BookImpl) GetMarketPrice(side order.Side, tokenId string) (decimal.Decimal, error) {
	if side == order.BUY {
		return b.GetBestBid(tokenId)
	} else {
		return b.GetBestAsk(tokenId)
	}
}
```

Reply:
gotcha but a market order could be for a sell too right? in which case this wouldn't work. Also this doesn't include depth. I think the TS version of this is flawed too and I think the FE passes a price which is why we don't have issues with it. I think what should happen here is if a price on the market order is not passed we should fetch the relevant book and calculate the weighted price for the size and then use that price to create the market order.